<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Prakat-D-013" name="Default test(failed)" tests="2" failures="1" timestamp="4 Jul 2019 10:38:17 GMT" time="0.003" errors="0">
  <testcase name="test3" time="0.001" classname="demo.TestNGRetryfailedDemo">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at demo.TestNGRetryfailedDemo.test3(TestNGRetryfailedDemo.java:29)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase name="test2" time="0.0" classname="demo.TestNGRetryfailedDemo"/>
</testsuite> <!-- Default test(failed) -->
